frontend/src/components/Psychologist/Reimbursement/BillingTable.jsx

```diff
@@ -3,37 +3,59 @@ import React from 'react';
 
 import { formatDDMMYYYY, utcDate } from 'services/date';
 
-const BillingTable = ({ filteredDate, appointments }) => {
+const boldContent = content => (
+  <div style={{ fontWeight: 'bold' }}>
+    {content}
+  </div>
+);
+const BillingTable = ({ filteredFirstDates, firstAppointments, filteredDates, appointments }) => {
   const columns = [
     {
       name: 'date',
       label: 'Date des séances effectuées',
-      render: date => (date === 'total' ? 'Total' : formatDDMMYYYY(utcDate(date))),
+      render: date => (date === 'total' ? boldContent('Total') : formatDDMMYYYY(utcDate(date))),
     },
     {
       name: 'appointment',
       label: 'Nombre de séances réalisées',
-      render: date => (date === 'total'
-        ? filteredDate.reduce((accumulator, d) => accumulator + appointments[d], 0)
-        : appointments[date]),
+      render: date => (date === 'total' ? boldContent(filteredDates.reduce((accumulator, d) => accumulator + appointments[d], 0)) : appointments[date]),
+    },
+    {
+      name: 'firstAppointment',
+      label: 'Dont premières séances',
+      render: date => (date === 'total' ? boldContent(filteredFirstDates.reduce((accumulator, d) => accumulator + firstAppointments[d], 0)) : firstAppointments[date] || 0),
     },
     {
       name: 'total',
       label: 'Total TTC €',
-      render: date => (date === 'total'
-        ? `${filteredDate.reduce((accumulator, d) => accumulator + appointments[d], 0) * 30}€`
-        : `${appointments[date] * 30}€`),
+      render: date => {
+        let nbAppointments = 0;
+        let nbFirstAppointments = 0;
+        if (date === 'total') {
+          nbFirstAppointments = filteredFirstDates.reduce((accumulator, d) => accumulator + firstAppointments[d], 0);
+          nbAppointments = filteredDates.reduce((accumulator, d) => accumulator + appointments[d], 0);
+        } else {
+          nbFirstAppointments = firstAppointments[date] || 0;
+          nbAppointments = appointments[date];
+        }
+
+        const totalAppointments = (nbAppointments - nbFirstAppointments) * 30;
+        const totalFirstAppointments = nbFirstAppointments * 40;
```

Comment:
:green_circle: On peut extraire ces magic numbers dans des constantes : 
```suggestion
        const FIRST_APPOINTMENT_TTC = 40;
        const NEXT_APPOINTMENT_TTC = 30;
        const totalAppointments = (nbAppointments - nbFirstAppointments) * NEXT_APPOINTMENT_TTC;
        const totalFirstAppointments = nbFirstAppointments * FIRST_APPOINTMENT_TTC;
```

(Je les met au dessus pour l'exemple, mais elles peuvent être mise en haut du fichier)